constants/evaa.py

```diff
@@ -0,0 +1,11 @@
+from constants.integration_token import Token
+
+EVAA_USDE_START_BLOCK = 52132128 # TODO: Replace with the actual start block number
```

Comment:
Here I think we can use a Ethereum mainnet block as a reference for the test since we will pass a simple timestamp to your API and otherwise it will fail with the current setup as we're using the a ETH provider to calculate the timestamp.

```suggestion
EVAA_USDE_START_BLOCK=22382834
```